### PR TITLE
Fix memory issue by applying incremental block file loading

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix memory issue by applying incremental block file loading
- Change ON_MEMORY_CHAIN_LENGTH: 20 -> 10

Related issues: https://github.com/ainblockchain/ain-blockchain/issues/290

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -523,28 +525,67 @@ class BlockchainNode {
...
-        process.exit(1); // NOTE(liayoo): Quick fix for the problem. May be fixed by deleting the block files.
+      logger.info(`[${LOG_HEADER}] Successfully loaded block: ${block.number} / ${block.epoch}`);
+      if (!Blockchain.validateBlock(block, prevBlockNumber, prevBlockHash)) {
+        logger.error(`[${LOG_HEADER}] Failed to validate block of number ${number}.`);
+        // NOTE(liayoo): Quick fix for the problem. May be fixed by deleting the block files.
+        process.exit(1);
       }
-      this.tp.cleanUpForNewBlock(block);
+      // NOTE(minsulee2): Deal with the case the only genesis block was generated.
+      if (!isFirstNode && number === numBlockFiles - 1) {
```
Comment (liayoo) [inline node/index.js:575]: I think the previous condition didn't require `!isFirstNode`. Is this intended? 
([previous logic](https://github.com/ainblockchain/ain-blockchain/pull/516/files#diff-8d14836825f4345ef7d6de4f4f623f33d6e1b7aac52d6faf27123044ff4524cdL58): `newChain && newChain.length > 1`)

Comment (platfowner) [reply node/index.js]: Right, just fixed by using wasBlockDirEmpty and isGenesisStart variables. Thanks!

**Comment by platfowner** [comment]
All comments addressed. PTAL @liayoo

File: blockchain/index.js [hunk 2]
```diff
@@ -151,56 +153,58 @@ class Blockchain {
...
   }
 
-  static isValidChain(chain, latestSnapshotBlockNumber) {
-    if (!chain.length) {
-      return true;
-    }
-    const firstBlock = Block.parse(chain[0]);
-    if (!firstBlock) {
+  static validateBlock(block, prevBlockNumber = null, prevBlockHash = null) {
+    const LOG_HEADER = 'loadAndExecuteChainOnDb';
```
Comment (liayoo) [inline blockchain/index.js:166]: validateBlock?

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks!!

File: blockchain/index.js (hunk 2 above)
Comment (platfowner) [reply blockchain/index.js]: Fixed. Thanks!

**Pull request merged**